### ISSUE TITLE
test against various PARI versions and with 32 bits

### ISSUE THREAD

**Issue by videlec** [defeo/cypari2#29]
We shoud test cypari2 against various pari versions

- the one shipped in Ubuntu on travis (will probably be impossible without using Docker since Ubuntu 14.04 is at pari 2.5.5)
- various master versions at http://pari.math.u-bordeaux.fr/download.html
- the latest snapshots. Their list can be obtained via
```
wget --quiet --output-document=- http://pari.math.u-bordeaux.fr/pub/pari/snapshots/ | grep -o "href=\"pari.*tar.gz\"" | sed "s/href=\"\(.*\)\"/\1/"
```

**Comment by jdemeyer** [comment]
> the one shipped in Ubuntu from travis

Do you happen to know which version that is?

> might be impossible because cypari2 currently relies on the header file pari/paripriv.h

Why should that matter? That header file is still installed like the other PARI header files.

**Comment by videlec** [comment]
>>  the one shipped in Ubuntu from travis

> Do you happen to know which version that is?

Sadly a very old one [pari 2.5.5](https://packages.ubuntu.com/en/trusty/libpari-dev) (travis is using Ubunutu 14.04 trusty)

**Comment by jdemeyer** [comment]
> Sadly a very old one pari 2.5.5 (travis is using Ubunutu 14.04 trusty)

That is never going to work...

**Comment by jdemeyer** [comment]
cypari2 needs at least PARI-2.8.0 because it uses the `cb_pari_err_handle` callback which was added only in version 2.8.0 (after much nagging by me in Bordeaux to convince the PARI developers that we really needed this)

**Comment by videlec** [comment]
fixed by #31
